src/formatDate.js

```diff
@@ -50,7 +50,40 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const parts = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
`fromFormat[fromFormat.length - 1]` looks a bit clumsy
you may get separator and other keys in this way: `const [...fromFormatKeys, separator] = fromFormat;`

Reply:
I get `A rest element must be last in a destructuring pattern.ts(2462) Parsing error: Rest element must be last element.` with your approach. I have another idea.